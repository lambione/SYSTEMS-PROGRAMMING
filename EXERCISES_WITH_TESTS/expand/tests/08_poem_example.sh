#!/bin/bash
exec "$1" tests/poem_rules.txt <<EOF
Ciao darkness, my old friend... should be: ciao darkness, my old friend...

Here is a poem by Blake.

Notice the expansion.

And here's another example of an expansion of empty text that you
should not see.  Well, you should see two consecutive spaces...
EOF
