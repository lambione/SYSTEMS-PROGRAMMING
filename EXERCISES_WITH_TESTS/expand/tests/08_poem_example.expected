Ciao darkness, my old friend... should be: hello darkness, my old friend...

Here is a Tyger Tyger, burning bright,
In the forests of the night;
What immortal hand or eye,
Could frame thy fearful symmetry?
 by Blake.

Notice the expansion.

And here's another example of an expansion of  that you
should not see.  Well, you should see two consecutive spaces...
